docs/mmds.md

```diff
@@ -2,13 +2,17 @@
 
 The Firecracker microVM Metadata Service (MMDS) is a mutable data store which
 implements a simplified data path, made possible by the unique setting found in
-Firecracker. The MMDS consists of three major logical components: the backend,
-the data store, and the minimalist HTTP/TCP/IPv4 stack (named *Dumbo*). They
-all exist within the Firecracker process, and outside the KVM boundary; the
-first is a part of the API server, the data store is a global entity for a
-single microVM, and the last is a part of the device model.
-When the API server is disabled by passing `--no-api` parameter to Firecracker, 
-MMDS is no longer available. 
+Firecracker. Software running inside the microVM can query the MMDS via HTTP
```

Comment:
I suggest to include the required guest network configuration steps that are required for the MMDS service to work with the new IP: add a route for the device for which a specific MMDS IP was configured.

Reply:
+1. Community has stumbled upon some confusion as far as using mmds. This is how #1226 got born. It would be useful to give out a full example of how mmds gets configured (even if that means simply describe in words what [this test](https://github.com/firecracker-microvm/firecracker/pull/1324/files#diff-bde8a6f9ba6139e13823be91dd973d24R187-R190) does.